src/content/fr/2019/resource-hints.md

```diff
@@ -0,0 +1,250 @@
+
+<figure>
+<div class="code-block floating-card">
+  <pre><code>&lt;carousel>
+  &lt;img src="cat1.jpg" importance="high">
+  &lt;img src="cat2.jpg" importance="low">
+  &lt;img src="cat3.jpg" importance="low">
+&lt;/carousel></code></pre></div>
+<figcaption>Figure 5. Exemple HTML d'utilisation d'indices de priorités sur un carrousel d'images.</figcaption>
+</figure>
+
+Par exemple, si vous disposez d'un carrousel d'images, des indices de priorités pourraient être utilisés pour prioriser l'image que les utilisateurs voient immédiatement et déprioriser les images ultérieures.
+
+<figure>
+  <div class="big-number">0,04 %</div>
+  <figcaption>Figure 6. Taux d'adoption des indices de priorités.</figcaption>
+</figure>
+
+Les indices de priorités sont [mis en œuvre](https://www.chromestatus.com/feature/5273474901737472) et peuvent être testés au moyen d'un drapeau de fonctionnalité dans les versions 70 et supérieures du navigateur Chromium. Étant donné qu'il s'agit encore d'une technologie expérimentale, il n'est pas surprenant qu'elle soit utilisée par 0,04 % des sites.
+
+85 % de l'utilisation des indices de priorités se fait avec les balises `<img>`. Les indices de priorités sont surtout utilisés pour déprioriser des ressources : 72 % de l'utilisation est `importance="low"` ; 28 % de l'utilisation est `importance="high"`.
+
+### Le <i lang="en">lazy loading</i> natif
+
+Le [<i lang="en">lazy loading</i> natif](https://web.dev/native-lazy-loading) est une API native permettant de différer le chargement des images et des iframes situées hors écran. Cela permet de libérer des ressources lors du chargement initial de la page et d'éviter de charger des ressources qui ne sont jamais utilisées. Auparavant, cette technique ne pouvait être réalisée qu'à l'aide de bibliothèques [JavaScript](./javascript) tierces.
+
+L'API pour le <i lang="en">lazy loading</i> natif se présente comme telle : `<img src="cat.jpg" loading="lazy">`.
+
+Le <i lang="en">lazy loading</i> natif est disponible dans les navigateurs basés sur le Chrome 76 et plus. L'API a été annoncée trop tard pour être incluse dans l'ensemble de données du Web Almanac de cette année, mais il faut la surveiller pour l'année prochaine.
+
+## Conclusion
+
+Dans l'ensemble, ces données semblent suggérer qu'il est pertinent de poursuivre l'adoption des indices de ressources. La plupart des sites gagneraient à adopter et / ou à passer de `dns-prefetch` à `preconnect`. Un sous-ensemble de sites beaucoup plus restreint pourrait bénéficier de l'adoption de la fonction `prefetch` et / ou `preload`. Utiliser avec succès `prefetch` et `preload` peut être plus délicat, ce qui tend à limiter leur adoption, mais les bénéfices potentiels sont également plus importants. Le HTTP/2 Push et la maturation des technologies d'apprentissage machine sont également susceptibles d'accroître l'adoption de `preload` et `prefetch`.
```

Comment:
Very minor nit, but the English version has a trailing new line and noticed this when comparing. So suggest we add this here, or (probably better) remove it from the English version.

```suggestion
Dans l'ensemble, ces données semblent suggérer qu'il est pertinent de poursuivre l'adoption des indices de ressources. La plupart des sites gagneraient à adopter et / ou à passer de `dns-prefetch` à `preconnect`. Un sous-ensemble de sites beaucoup plus restreint pourrait bénéficier de l'adoption de la fonction `prefetch` et / ou `preload`. Utiliser avec succès `prefetch` et `preload` peut être plus délicat, ce qui tend à limiter leur adoption, mais les bénéfices potentiels sont également plus importants. Le HTTP/2 Push et la maturation des technologies d'apprentissage machine sont également susceptibles d'accroître l'adoption de `preload` et `prefetch`.

```